src/odb/src/swig/tcl/CMakeLists.txt

```diff
@@ -5,6 +5,7 @@ swig_lib(NAME          odbtcl
          I_FILE        ../common/odb.i
          SWIG_INCLUDES ${PROJECT_SOURCE_DIR}/include
                        ${PROJECT_SOURCE_DIR}/src/swig/tcl
+         DEFINES       TEST_BUILD
```

Comment:
What is the purpose of TEST_BUILD?

Reply:
I wasn't able to generate a stub correctly, but I just needed a nights sleep. This parameter is gone now.